sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.createOffering.e2e-spec.ts

```diff
@@ -57,14 +68,46 @@ describe("EducationProgramOfferingInstitutionsController(e2e)-createOffering", (
       db.dataSource,
       InstitutionTokenTypes.CollegeFUser,
     );
+    // Get BC Private institution and institution user.
+    const { institution: collegeC, user: collegeCUser } =
+      await getAuthRelatedEntities(
+        db.dataSource,
+        InstitutionTokenTypes.CollegeCUser,
+      );
+    // Get out of province institution and institution user.
+    const { institution: collegeD, user: collegeDUser } =
+      await getAuthRelatedEntities(
+        db.dataSource,
+        InstitutionTokenTypes.CollegeDUser,
+      );
+    institutionCollegeC = collegeC;
+    institutionCollegeCUser = collegeCUser;
+    institutionCollegeD = collegeD;
+    institutionCollegeDUser = collegeDUser;
     collegeF = institution;
     collegeFUser = institutionUser;
     collegeFLocation = createFakeInstitutionLocation({ institution: collegeF });
+    collegeCLocation = createFakeInstitutionLocation({
+      institution: institutionCollegeC,
+    });
+    collegeDLocation = createFakeInstitutionLocation({
+      institution: institutionCollegeD,
+    });
     await authorizeUserTokenForLocation(
       db.dataSource,
       InstitutionTokenTypes.CollegeFUser,
       collegeFLocation,
     );
+    await authorizeUserTokenForLocation(
```

Comment:
Not a blocker for E2Es, but these ones would be great candidates to be executed in parallel.